x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/models_management/expanded_row.tsx

```diff
@@ -104,8 +106,8 @@ export const ExpandedRow: FC<ExpandedRowProps> = ({ item }) => {
             ),
         };
       })
-      .filter(({ description }) => {
-        return description !== undefined;
+      .filter(({ description: d }) => {
+        return d !== undefined;
```

Comment:
Tiny nit: could be implicit return.

Reply:
do you mean ` .filter(({ description: d }) => d !== undefined)`?